components/table/table-wrapper.js

```diff
@@ -54,10 +87,46 @@ export const tableStyles = css`
 		font-size: 0.7rem;
 		line-height: 0.9rem;
 	}
-	d2l-table-wrapper[type="default"] .d2l-table > thead > tr > th,
-	d2l-table-wrapper[type="default"] .d2l-table > * > tr.d2l-table-header > th,
-	d2l-table-wrapper[type="default"] .d2l-table > * > tr[header] > th {
-		height: 27px; /* min-height to be 48px including border */
+
+	d2l-table-col-sort-button {
+		vertical-align: middle;
+	}
+	d2l-table-col-sort-button:not(:only-child) {
+		--d2l-table-col-sort-button-additional-padding-inline-end: 0px; /* stylelint-disable-line length-zero-no-unit */
+	}
+	d2l-table-col-sort-button:not(:only-child):first-child {
+		--d2l-table-col-sort-button-margin-inline-end: 1px;
```

Comment:
Minimize space between `d2l-table-col-sort-button`s on siblings (default is 4px)